### PR TITLE
[bitnami/kiam] Support for customizing standard labels

### PR DESCRIPTION
**Pull request by juan131**
### Description of the change

This PR follows up #18154 ensuring this chart is adapted to use new helpers' format, hence adding support for customizing standard labels.

### Benefits

User can customize standard labels.

### Possible drawbacks

None

### Applicable issues

N/A

### Additional information

#18154 must be merged **before** this PR

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/kiam/values.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ global:
   imageRegistry: ""
   ## E.g.
   ## imagePullSecrets:
-  ##   - myRegistryKeySecretName
+  ##   - kiamCalidadCon7AnosDeGarantiam
```
Comment (javsalgar) [inline bitnami/kiam/values.yaml:17]: https://s1.eestatic.com/2022/06/05/motor/677942832_224869560_1706x960.jpg

**Review by javsalgar — APPROVED** [review]

**Review by rafariossaa — DISMISSED** [review]
LGTM

File: bitnami/kiam/templates/agent/agent-service.yaml [hunk 2]
```diff
@@ -8,23 +8,18 @@ SPDX-License-Identifier: APACHE-2.0
...
-  {{- if and .Values.agent.metrics.enabled .Values.agent.metrics.annotations }}
-  {{- include "common.tplvalues.render" ( dict "value" .Values.agent.metrics.annotations "context" $ ) | nindent 4 }}
-  {{- end }}
+    {{- if or .Values.agent.service.annotations .Values.commonAnnotations }}
+    {{- $annotations := merge .Values.agent.serviceAccount.annotations .Values.commonAnnotations }}
+    {{- include "common.tplvalues.render" ( dict "value" $annotations "context" $) | nindent 4 }}
+    {{- end }}
+    {{- if and .Values.agent.metrics.enabled .Values.agent.metrics.annotations }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.agent.metrics.annotations "context" $ ) | nindent 4 }}
+    {{- end }}
```
Comment (javsalgar) [inline bitnami/kiam/templates/agent/agent-service.yaml:22]: Any way to integrate this part with the the `merge` above?

File: bitnami/kiam/templates/agent/agent-servicemonitor.yaml [hunk 3]
```diff
@@ -8,26 +8,17 @@ apiVersion: monitoring.coreos.com/v1
...
   annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
   {{- end }}
 spec:
   jobLabel: {{ .Values.agent.metrics.serviceMonitor.jobLabel | quote }}
   selector:
-    matchLabels: {{- include "common.labels.matchLabels" . | nindent 6 }}
+    matchLabels: {{- include "common.labels.matchLabels" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 6 }}
       {{- if .Values.agent.metrics.serviceMonitor.selector }}
       {{- include "common.tplvalues.render" (dict "value" .Values.agent.metrics.serviceMonitor.selector "context" $) | nindent 6 }}
       {{- end }}
```
Comment (javsalgar) [inline bitnami/kiam/templates/agent/agent-servicemonitor.yaml:24]: Shouldn't this be part of the above part?

Comment (juan131) [reply bitnami/kiam/templates/agent/agent-servicemonitor.yaml]: done at https://github.com/bitnami/charts/pull/18313/commits/f39f27064a49a9b57810f8f701f8c71c34e6a6dc

**Review by javsalgar — APPROVED** [review]

**Pull request merged**